tools/nme/src/platforms/IOSPlatform.hx

```diff
@@ -327,6 +329,22 @@ ${hxcpp_include}';
       //updateLaunchImage();
    }
 
+   private function getAbsolutePath(binary:String): String {
+       var process:Process;
+       #if windows
```

Comment:
Shouldn't that be a runtime check? Otherwise it only works when it was _compiled_ on the right OS..

Reply:
Besides, looks like you're doing the same thing in both branches. :)

Reply:
Good catch, updated to use runtime check for OS, and using `where.exe`for windows.